Classes/Command/FakeStorageCommandController.php

```diff
@@ -215,8 +215,8 @@ public function ListStoragesCommand(): void
         $this->output->output('uid   |   name   |   driver   |   status' . PHP_EOL);
 
         foreach ($storages as $storage) {
-            $isFakeStorage = $storage['tx_fakefal_enable'] === 1 ? 'fake-storage' : 'real storage';
-            $this->output->output($storage['uid'] . '   |   ' . $storage['name'] . '   |   ' . $storage['driver'] . '   |   ' . $isFakeStorage . '   |   ' . PHP_EOL);
+            $status = $storage['tx_fakefal_enable'] === 1 ? 'fake-storage' : 'real storage';
```

Comment:
I would remove this `$status` variable, see comment in the other commit; imho much cleaner with a corresponding table header and `true`/`false` values in the column.